rfcs/20200916-sig-tf-js.md

```diff
@@ -0,0 +1,107 @@
+- Driving TensorFlow.js features for Server-side execution with tfjs-node. This will include 
+building tooling and infra for end to end, production-ready ML with TensorFlow.js. 
+- Developing features for specific platforms and deployments (eg. React Native, IoT and embedded 
+devices, etc); 
+- Building workflows for Transfer learning features for customizing models
+- Adding support for newer models and application areas (eg. NLP, RL, etc), and data analysis 
+algorithms
+- Conducting research and implementing prototypes for client-side, privacy preserving ML and 
+Federated Learning. 
+- Identifying performance issues, tradeoffs, and requirements.
+- Model benchmarking across browsers, hardware and devices.
+- Model encryption.
+- Model visualization.
+- These projects will move towards community ownership as work streams mature. 
+
+
+## Membership
+
+We encourage any developers working at the intersection of machine learning and web/JS 
+applications to join and participate in the activities of the SIG. Whether you are working on 
+advancing the platform, prototyping or building specific applications, or authoring new libraries, 
+we welcome your feedback on and contributions to Tensorflow.js and its tooling, and are eager to hear 
+about any downstream results, implementations, and extensions. 
+
+We have multiple channels for participation, and publicly archive discussions in our user group and 
+announcements mailing lists:
+- tfjs@tensorflow.org -- our general mailing list that all are welcome to join ([archive group](https://groups.google.com/a/tensorflow.org/g/tfjs))
+- tfjs-announce@tensorflow.org -- Announcement only mailing list for TF.js community ([archive group](https://groups.google.com/a/tensorflow.org/g/tfjs-announce))
+
+We will create a new mailing list for TensorFlow.js SIG members. 
+
+### Other Resources 
+- Repository: https://github.com/tensorflow/tfjs
+- Documentation: https://www.tensorflow.org/js
+
+## Organization and Governance
+SIG project leads will be added to a SIG TF.js maintainers team on GitHub to streamline their 
+contributions to tensorflow/tfjs. As specific ideas and work streams develop, we will explore creating 
+new, community-owned repos that the SIG will drive.
+
+## Contacts
+### Project Lead(s):
+- Ping Yu (Google)
+- Sandeep Gupta (Google)
+- Ton Ngo (IBM): 
+  - Server side support (GPU, training)
+  - End-to-end pipeline on Kubernetes (Kubeflow)
+- Ricky Cao (Alibaba)
+  - Building tooling and infra for end to end, production-ready ML with TensorFlow.js.
+  - Pipeline with transfer learning
+  - Model Encryption
+- Ningxin Hu (Intel)
+  - Benchmarking and performance optimization
+- Gant Laborde (Infinite Red)
+  - Platforms (React and React Native)
+  - Adding support for newer models and application areas (eg. NLP, RL, etc),
+  - Model visualization
+
+### SIG members: 
+- Eric Li (Alibaba)
+- Zebai (Alibaba)
+- Paul Van Eck (IBM)
+- Ted Chang (IBM)
+- YiHong Wang (IBM)
+
```

Comment:
`- Saswat Samal (Student)`

Looking forwards to join this awesome group of TF and TFJs Developers to learn more about TF and TFJs!